sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxDocumentClientImpl.java

```diff
@@ -2586,7 +2589,9 @@ private Mono<ResourceResponse<Document>> createDocumentInternal(
                         return Mono.error(new IllegalStateException("documentCollectionValueHolder or documentCollectionValueHolder.v cannot be null"));
                     }
 
-                    return this.partitionKeyRangeCache.tryLookupAsync(BridgeInternal.getMetaDataDiagnosticContext(request.requestContext.cosmosDiagnostics), documentCollectionValueHolder.v.getResourceId(), null, null)
+                    StringBuilder sb = new StringBuilder("RxDocumentClientImpl.createDocumentInternal:").append(",");
```

Comment:
I do not think we want to check in the log change in master, maybe have one PR only includes the defensive mechanism change?